products-service/src/functions/getProductsById/handler.ts

```diff
@@ -1,19 +1,31 @@
 import 'source-map-support/register';
 
+import { Pool } from 'pg';
+import dbConfig from '../../dbConfig';
 import { middyfy } from '../../libs/lambda';
 import { formatJSONResponse } from '../../libs/apiGateway';
 
-import { ProductsRepo } from '../../shared/ProductsRepo';
+let pool;
 
 const getProductsById = async (event) => {
+  if (!pool) {
+    pool = new Pool(dbConfig);
+  }
+
+  const client = await pool.connect();
+
   try {
     const { productId } = event.pathParameters;
```

Comment:
you should take productId from `queryStringParameters`, as your url is 
`https://sdyaa5isjk.execute-api.eu-west-1.amazonaws.com/dev/products?productId=someId`
if you want to use `pathParameters `, then the url should be
`https://sdyaa5isjk.execute-api.eu-west-1.amazonaws.com/dev/products/someId`

Reply:
> you should take productId from `queryStringParameters`, as your url is
> `https://sdyaa5isjk.execute-api.eu-west-1.amazonaws.com/dev/products?productId=someId`
> if you want to use `pathParameters `, then the url should be
> `https://sdyaa5isjk.execute-api.eu-west-1.amazonaws.com/dev/products/someId`

Sorry, wrong link in the description. Updated.